LanguageFeatures/Parts-with-imports/scope_A08_t02_part1.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It’s a compile-time error if any file declares an import prefix
+/// with the same base name as a top-level declaration of the library.
+///
+/// @description Check that it is a compile-time error if a part-file declares
+/// an import prefix with the same base name as a top-level declaration.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=enhanced-parts
+
+part of 'scope_A08_t01.dart';
+
+import 'scope_lib2.dart' as p1;
+import 'scope_lib2.dart' as p2;
+import 'scope_lib2.dart' as p3;
+import 'scope_lib2.dart' as p4;
+import 'scope_lib2.dart' as p5;
+import 'scope_lib2.dart' as p6;
+import 'scope_lib2.dart' as p7;
+import 'scope_lib2.dart' as p8;
+import 'scope_lib2.dart' as p9;
+
+part 'scope_A08_t01_part2.dart';
+
+int l1 = 1;
```

Comment:
Oh, but then we still have the top-level name clash, so we won't know for sure whether the conflict between the top-level names and the import prefixes is detected.

Reply:
Ah! `scope_A08_t02_part1.dart` and  `scope_A08_t02_branch2_part2.dart` both adding `lx` declarations in a top-level scope and therefore it is a name conflict! Of course! Thank you for noticing! I'll fix this in this PR and add a separate test(s) to check it!

Reply:
Great, thanks!